rachellegelden/Home-Together-Canada
# HTC-446: Load Business Info Into Account Summary

# [HTC-446](https://github.com/rachellegelden/Home-Together-Canada/issues/446)

## Summary
- loaded business info into Business Account Info subpage of the account summary
- refactored BusinessAccountSummary
- solved console errors that appeared in the account summary and business registration
- added support for fetching images to the server

## Relevant Motivation & Context
So that businesses can view their currently saved account info

## Testing Instructions
1. register a business user (with a logo)
2. go to Account Info and ensure that all information is displayed properly
3. change the logo image, the old logo should be hidden
4. repeat process above but when registering a business without a logo

## ⚠️ NOTES ⚠️ 
- You cannot update the account info as of right now
- Tests for the helper functions I wrote will be created in the next PR (where I implement the update functionality)

## Developer checklist prior to opening this pull request:


## Reviewer
